introduction.md

```diff
@@ -0,0 +1 @@
+My name is Nicolas the Pretty Good, but you can call Nicolas the Pretty Good. I am a developer stationed in Miami, soon to move to San Francisco to dwelve deeper into the cyber tribe. I have a background in psychology, which leads me to often reflect on the similarities between the mind and programs, brains and computers. I love to create, solve problems, read, learn, humor in general. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.